src/components/com_tjcertificate/administrator/models/forms/filter_certificates.xml

```diff
@@ -31,16 +31,16 @@
 			<option value="">COM_TJCERTIFICATE_CERTIFICATE_FILTER_CERTIFICATE_CLIENT_SELECT</option>
 		</field>
 
-		<!-- <field
+		<field
```

Comment:
I think you will need a custom field here because this will only give you users who have Joomla account but not the client_issue_to column'd data